Dockerfile

```diff
@@ -5,9 +5,15 @@ ENV LANG C.UTF-8
 
 WORKDIR /docs-scraper
 
-RUN apt-get update -y && apt-get install -y python3-pip libnss3 chromium-driver
+RUN : \
```

Comment:
What is this `:` notation in this line `RUN : \` @sanders41? I never saw that!

Reply:
It just let me split it across multiple lines from the start.

Reply:
nice, cool!! :D